include/Transforms/UnusedMemRef/UnusedMemRef.td

```diff
@@ -0,0 +1,17 @@
+#ifndef INCLUDE_TRANSFORMS_UNUSEDMEMREF_UNUSEDMEMREF_TD_
+#define INCLUDE_TRANSFORMS_UNUSEDMEMREF_UNUSEDMEMREF_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def UnusedMemRef : Pass<"unused-memref", "func::FuncOp"> {
```

Comment:
nit: might be better to say what it does with unused memrefs in the pass name (`remove-unused-memref` perhaps)

Reply:
done!